_generator/types.yaml

```diff
@@ -87,7 +87,7 @@ types:
     file: accountnotes.md
     anchor: account-note-update-classifications
   unit:
-    file: _objects.md
+    file: vouchers.md
```

Comment:
Why did this change?

Reply:
This is a bug in generator, unit behaves strangely since it's a named type but doesn't have any properties, so for some reason the generator always overrides it. 🤔